gym/spaces/multi_binary.py

```diff
@@ -58,7 +58,7 @@ def sample(self, mask: Optional[np.ndarray] = None) -> np.ndarray:
 
         Args:
             mask: An optional np.ndarray to mask samples with expected shape of ``space.shape``.
-                Where mask == 0 then the samples will be 0.
+                Where for each binary action, it expects shape mask in ``dtype=np.int8``.
```

Comment:
Can we keep both pieces of information in the docstring?

Reply:
I discovered a bug here as the current sampling allows for the mask shape == space shape and mask value allow `0 -> 0` and `1 -> random 0 or 1` so doesn't allow the mask to force `1` as only value answer. The easiest solution is to add `1 -> 1` and `2 -> random 0 or 1` but this breaks convention. The other option is making the mask shape == space shape + (2,).